### PR TITLE
AAP-29213 - Convert authentication type content

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR converts the content for the authentication type modules from the Google draft to AsciiDoc.

https://docs.google.com/document/d/1mEYguv3V0WUUHBMc8tsgqlK8RB3ro8IZSlQmMuFfhww/edit#bookmark=id.85uao3v7ar2t

### CODE REVIEW

File: downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc [hunk 1]
```diff
@@ -4,36 +4,33 @@
...
-
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
+Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends. The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
 
 .Procedure
-. From the navigation panel, select {MenuAEAdminSettings}.
-. On the *Settings* page, select *GitHub settings* from the list of *Authentication* options.
-. Click the *GitHub Enterprise Organization* tab.
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc:14]: I think I'm missing a step here: 
```suggestion
. From the navigation panel, select {MenuAMAuthentication}.
. Click btn:[Create authentication].
```

File: downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc [hunk 2]
```diff
@@ -4,36 +4,33 @@
...
+. Select *GitHub enterprise organization* from the *Authentication type* list.
+. Enter a *Name* for this authentication configuration.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc:16]: Maybe this is too much info. Just adding a suggestion.
```suggestion
. Enter a *Name* for this authentication configuration and click btn:[Next].
```

File: downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc [hunk 3]
```diff
@@ -4,36 +4,33 @@
...
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
+Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends. The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
 
 .Procedure
-. From the navigation panel, select {MenuAEAdminSettings}.
-. On the *Settings* page, select *GitHub settings* from the list of *Authentication* options.
-. Click the *GitHub Enterprise Organization* tab.
+. From the navigation panel, select {MenuAMAuthentication}.
+. Select *GitHub enterprise organization* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc:15]: Suggestion based on UI:
```suggestion
. Select *GitHub enterprise organization* from the *Authentication type* list and click btn:[Next].
```

File: downstream/modules/platform/proc-controller-github-enterprise-settings.adoc [hunk 4]
```diff
@@ -1,37 +1,35 @@
...
-
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
+Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends. The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
 
 .Procedure
-. From the navigation panel, select {MenuAEAdminSettings}.
-. On the *Settings* page, select *GitHub settings* from the list of *Authentication* options.
-. Click the *GitHub Enterprise* tab.
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-settings.adoc:14]: Suggestion:
```suggestion
. From the navigation panel, select {MenuAMAuthentication}.
. Click btn:[Create authentication].
```

File: downstream/modules/platform/proc-controller-github-enterprise-settings.adoc [hunk 5]
```diff
@@ -1,37 +1,35 @@
...
+. Select *GitHub enterprise* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-settings.adoc:15]: Suggestion:
```suggestion
. Select *GitHub enterprise* from the *Authentication type* list and click btn:[Next].
```

File: downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc [hunk 6]
```diff
@@ -1,41 +1,40 @@
...
 The OAuth2key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
 
 .Procedure
-. Find the numeric team ID using the link:https://fabian-kostadinov.github.io/2015/01/16/how-to-find-a-github-team-id/[GitHub API].
-The Team ID will be used to supply a required field in the UI.
-. From the navigation panel, select {MenuAEAdminSettings}.
-. On the *Settings* page, select *GitHub settings* from the list of *Authentication* options.
-. Click the *GitHub Enterprise Team* tab.
+
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc:18]: Globally, suggesting this update. 
```suggestion
. From the navigation panel, select {MenuAMAuthentication}.
. Click btn:[Create authentication].
```

File: downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc [hunk 7]
```diff
@@ -1,41 +1,40 @@
...
+. Select *GitHub enterprise team* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc:19]: Global suggestion for this step: 
```suggestion
. Select *GitHub enterprise team* from the *Authentication type* list and click btn:[Next].
```

File: downstream/modules/platform/proc-controller-github-organization-setttings.adoc [hunk 8]
```diff
@@ -1,37 +1,36 @@
...
 
-= GitHub Organization settings
-
-When defining account authentication with either an organization or a team within an organization, you should use the specific organization and team settings.
-Account authentication can be limited by an organization and by a team within an organization.
+= Configuring GitHub organization authentication
 
+When defining account authentication with either an organization or a team within an organization, you should use the specific organization and team settings. Account authentication can be limited by an organization and by a team within an organization.
 You can also choose to permit all by specifying non-organization or non-team based settings.
+You can limit users who can login to the platform by limiting only those in an organization or on a team within an organization.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-organization-setttings.adoc:7]: Tiny nit: 
```suggestion
You can limit users who can log in to the platform by limiting only those in an organization or on a team within an organization.
```

File: downstream/modules/platform/proc-controller-github-organization-setttings.adoc [hunk 9]
```diff
@@ -1,37 +1,36 @@
...
-To register the application, you must supply it with your Authorization callback URL, which is the *Callback URL* shown in the *Details* page.
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI.
+The OAuth2 key (Client ID) and secret (Client Secret) are used to supply the required fields in the UI. To register the application, you must supply it with your webpage URL, which is the Callback URL shown in the Authenticator details for your authenticator configuration. See xref:gw-display-auth-details[Displaying authenticator details] for instructions on accessing this information.
 
 .Procedure
-. From the navigation panel, select {MenuAEAdminSettings}.
-. On the *Settings* page, select *GitHub settings* from the list of *Authentication* options.
-. Select the *GitHub Organization* tab.
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-organization-setttings.adoc:14]: See the global update for this in above sections. (Need to add "Click **Create authentication**."

File: downstream/modules/platform/proc-controller-github-organization-setttings.adoc [hunk 10]
```diff
@@ -1,37 +1,36 @@
...
+. Select *GitHub organization* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-organization-setttings.adoc:15]: See global suggestion for this step.

File: downstream/modules/platform/proc-controller-github-settings.adoc [hunk 11]
```diff
@@ -1,28 +1,29 @@
...
-. Select the *GitHub Default* tab if not already selected.
-+
-The *GitHub OAuth2 Callback URL* field is already pre-populated and non-editable.
-When the application is registered, GitHub displays the Client ID and Client Secret.
 
-. Click btn:[Edit] and copy and paste the GitHub Client ID into the *GitHub OAuth2 Key* field.
-. Copy and paste the GitHub Client Secret into the *GitHub OAuth2 Secret* field.
-. For more information on completing the mapping fields, see xref:ref-controller-organization-mapping[Organization mapping] and xref:ref-controller-team-mapping[Team mapping].
-. Click btn:[Save].
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-settings.adoc:11]: See global update that follows this step above.

File: downstream/modules/platform/proc-controller-github-settings.adoc [hunk 12]
```diff
@@ -1,28 +1,29 @@
...
+. Select *GitHub* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-settings.adoc:12]: See global suggestion for this step.

File: downstream/modules/platform/proc-controller-github-team-settings.adoc [hunk 13]
```diff
@@ -1,32 +1,34 @@
...
-The *GitHub Team OAuth2 Callback URL* field is already pre-populated and non-editable.
-When the application is registered, GitHub displays the Client ID and Client Secret.
 
-. Click btn:[Edit] and copy and paste GitHub's Client ID into the *GitHub Team OAuth2 Key* field.
-. Copy and paste GitHub's Client Secret into the *GitHub Team OAuth2 Secret* field.
-. Copy and paste GitHub's team ID in the *GitHub Team ID* field.
-. For more information on completing the mapping fields, see xref:ref-controller-organization-mapping[Organization mapping] and xref:ref-controller-team-mapping[Team mapping].
-. Click btn:[Save]
+. From the navigation panel, select {MenuAMAuthentication}.
+. Select *GitHub team* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-github-team-settings.adoc:14]: See global suggestion for this step.

File: downstream/modules/platform/proc-controller-google-oauth2-settings.adoc [hunk 14]
```diff
@@ -1,36 +1,43 @@
...
-. On the *Settings* page, select *Google OAuth 2 settings* from the list of *Authentication* options.
-+
-The *Google OAuth2 Callback URL* field is already pre-populated and non-editable.
-
-. The following fields are also pre-populated.
-If not, use the credentials Google supplied during the web application setup process, and look for the values with the same format as the ones shown in the example below:
 
-* Click *Edit* and copy and paste Google's Client ID into the *Google OAuth2 Key* field.
-* Copy and paste Google's Client secret into the *Google OAuth2 Secret* field.
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-google-oauth2-settings.adoc:13]: See global update that follows this step above.

File: downstream/modules/platform/proc-controller-set-up-azure.adoc [hunk 15]
```diff
@@ -1,35 +1,34 @@
...
 
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-To register the application, you must supply it with your webpage URL, which is the Callback URL shown in the *Authentication* tab of the *Settings* screen.
+Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends. To register the application, you must supply it with your webpage URL, which is the Callback URL shown in the Authenticator details for your authenticator configuration. See dxref:gw-display-auth-details[Displaying authenticator details] for instructions on accessing this information.  
 
 .Procedure
 
-. From the navigation panel, select {MenuAEAdminSettings}.
-. Select *Azure AD settings* from the list of *Authentication* options.
+. From the navigation panel, select {MenuAMAuthentication}.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-set-up-azure.adoc:11]: See global suggestions for this step in examples above. (includes Click **Create Authentication**.)

File: downstream/modules/platform/proc-controller-set-up-azure.adoc [hunk 16]
```diff
@@ -1,35 +1,34 @@
...
+. Select *Azuread* from the *Authentication type* list.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-controller-set-up-azure.adoc:12]: See global suggestions in examples above to include "Click **Next**."

**Comment by jself-sudoku** [comment]
Overall, LGTM. Just a few simple, global updates to apply.

File: downstream/modules/platform/proc-controller-deleting-a-user.adoc [hunk 17]
```diff
@@ -5,6 +5,7 @@
 = Deleting a user
 
 Before you can delete a user, you must have normal user or system administrator permissions. When you delete a user account, the name and email of the user are permanently removed from {PlatformNameShort}.
+
 .Procedure
 . From the navigation panel, select {MenuAMUsers}.
 . Select the check box for the user that you want to remove.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-deleting-a-user.adoc:11]: Good catch! FYI: The style guide says it should be one word. This is fixed.

File: downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc:14]: Fixed.

File: downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-enterprise-org-settings.adoc:15]: Fixed.

File: downstream/modules/platform/proc-controller-github-enterprise-settings.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-enterprise-settings.adoc:14]: Fixed.

File: downstream/modules/platform/proc-controller-github-enterprise-settings.adoc (hunk 5 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-enterprise-settings.adoc:15]: Fixed.

File: downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc (hunk 6 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc:18]: Fixed.

File: downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc (hunk 7 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-enterprise-team-settings.adoc:19]: Fixed.

File: downstream/modules/platform/proc-controller-github-organization-setttings.adoc (hunk 8 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-organization-setttings.adoc:7]: Fixed.

File: downstream/modules/platform/proc-controller-github-organization-setttings.adoc (hunk 9 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-organization-setttings.adoc:14]: Fixed.

File: downstream/modules/platform/proc-controller-github-organization-setttings.adoc (hunk 10 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-organization-setttings.adoc:15]: Fixed.

File: downstream/modules/platform/proc-controller-github-settings.adoc (hunk 12 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-settings.adoc:12]: Fixed.

File: downstream/modules/platform/proc-controller-github-team-settings.adoc [hunk 18]
```diff
@@ -1,32 +1,34 @@
...
-+
-The *GitHub Team OAuth2 Callback URL* field is already pre-populated and non-editable.
-When the application is registered, GitHub displays the Client ID and Client Secret.
 
-. Click btn:[Edit] and copy and paste GitHub's Client ID into the *GitHub Team OAuth2 Key* field.
-. Copy and paste GitHub's Client Secret into the *GitHub Team OAuth2 Secret* field.
-. Copy and paste GitHub's team ID in the *GitHub Team ID* field.
-. For more information on completing the mapping fields, see xref:ref-controller-organization-mapping[Organization mapping] and xref:ref-controller-team-mapping[Team mapping].
-. Click btn:[Save]
+. From the navigation panel, select {MenuAMAuthentication}.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-team-settings.adoc:13]: Fixed.

File: downstream/modules/platform/proc-controller-github-team-settings.adoc (hunk 13 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-github-team-settings.adoc:14]: Fixed.

File: downstream/modules/platform/proc-controller-google-oauth2-settings.adoc [hunk 19]
```diff
@@ -1,36 +1,43 @@
...
+. Click btn:[Create authentication].
+. Select *Google OAuth* from the *Authentication type* list.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-google-oauth2-settings.adoc:15]: Fixed.

File: downstream/modules/platform/proc-controller-set-up-azure.adoc (hunk 15 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-azure.adoc:11]: Fixed.

File: downstream/modules/platform/proc-controller-set-up-azure.adoc (hunk 16 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-azure.adoc:12]: Fixed.

File: downstream/modules/platform/proc-controller-set-up-generic-oidc.adoc [hunk 20]
```diff
@@ -1,34 +1,49 @@
...
+OpenID Connect (OIDC) uses the OAuth 2.0 framework. It enables third-party applications to verify the identity and obtain basic end-user information. The main difference between OIDC and SAML is that SAML has a service provider (SP)-to-IdP trust relationship, whereas OIDC establishes the trust with the channel (HTTPS) that is used to obtain the security token. To obtain the credentials needed to set up OIDC with {PlatformNameShort}, see the documentation from the IdP of your choice that has OIDC support.
 
 .Procedure
 
-. From the navigation panel, select {MenuAEAdminSettings}.
-. Select *Generic OIDC settings* from the list of *Authentication* options.
-. Click btn:[Edit] and enter the following information:
+. From the navigation panel, select {MenuAMAuthentication}.
+. Click btn:[Create authentication].
+. Select *Generic OIDC* from the *Authentication type* list.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-set-up-generic-oidc.adoc:11]: Fixed.

**Comment by jself-sudoku** [comment]
LGTM

**Pull request merged**